src/main/java/gighub/worketserver/domain/ContractModify.java

```diff
@@ -0,0 +1,40 @@
+package gighub.worketserver.domain;
+
+import gighub.worketserver.domain.constants.ModifyStatus;
+import jakarta.persistence.*;
+import lombok.*;
+
+import java.time.LocalDateTime;
+
+@Entity
+@Table(name = "contract_modify")
```

Comment:
클래스명은 단수로, DB에 저장될 때는 복수형으로 이렇게 수정하면 어떨까요?

```suggestion
@Table(name = "contract_modification_requests")
```

Reply:
네 !!! 그렇게 수정하겠습니다!!